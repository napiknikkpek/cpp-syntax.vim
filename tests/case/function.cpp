void foo();
foo(1);
foo::bar();
foo(a, b, c);
foo(A a, B b, C c);
foo<T>();
foo<decltype(a > b)>();
foo<std::vector<int>>();
foo<std::vector<decltype(a > b)>>();
foo<decltype(bar<T>())>();
