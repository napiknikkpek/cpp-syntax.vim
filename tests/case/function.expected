zzzz.fff...
fff.c..
ttt..fff...
fff.x..x..x..
fff.t.x..t.x..t.x..
fff.t....
fff.aaaaaaaa.x...x.....
fff.ttt..tttttt.zzz.....
fff.ttt..tttttt.aaaaaaaa.x...x......
fff.aaaaaaaa.fff.t........
